onnxruntime/core/mlas/lib/sqnbitgemm.cpp

```diff
@@ -496,23 +595,43 @@ InitializeWorkspace_CompInt8(
     MLAS_UNREFERENCED_PARAMETER(N);
 
     const auto QuantizeARow = GetMlasPlatform().SQNBitGemmDispatch->QuantizeARow_CompInt8;
+    const auto QuantizeARow2 = GetMlasPlatform().SQNBitGemmDispatch->QuantizeARow_CompInt8_2;
```

Comment:
>QuantizeARow_CompInt8_2 [](http://example.com/codeflow?start=69&length=23)

a better name, like QuantizeARowBlkSum?

Reply:
renamed to QuantizeARowComputeBlkSum_CompInt8